src/io.jl

```diff
@@ -0,0 +1,90 @@
+"""
+    GeoJSON.read(json::String; lazyfc=false, ndim=2)
+
+Read GeoJSON from a string or file to a GeoInterface.jl compatible object.
```

Comment:
I tried a file but it doesn't work yet since https://github.com/quinnj/JSON3.jl/pull/243 is not yet tagged.